.github/mergify.yml

```diff
@@ -0,0 +1,26 @@
+pull_request_rules:
+  - name: backport to galactic at reviewers discretion
+    conditions:
+      - base=main
```

Comment:
We could steal the [approach](https://github.com/ros-planning/navigation2/blob/main/.github/mergify.yml#L34) of @SteveMacenski

Reply:
Also, I thought deleting the branches would be default behavior. Do we need [this](https://github.com/ros-planning/navigation2/blob/main/.github/mergify.yml#L20) to clean up after merging?

Reply:
These won't affect since the base=main. I was gonna ask the same question, we may or may not want to forward port things from foxy/galactic to main. Mergify currently doesn't allow that with the current config, so should be good. I saw the approach but then thought especially with the current focus on Foxy tutorials we may want to forward port some fixes / changes from foxy to galactic and main. Let me know what do you think?

Reply:
I created this problem by making the foxy branch diverge from the main branch.  Because I can't really delete all the broken tutorials out of main (for political reasons mostly), I think we should port changes forward and allow development on the version-specific branches.

Reply:
Someday when the main branch of tutorials is cleaning working and doesn't contain dead code then I think we should change to only develop on main (as nav2 does).

Reply:
The main branch dev cycle is also representative of a shift in the core ROS 2  repos themselves with `ros2` and `main` or `rolling`. There's no more development in released distributions happening really anymore anywhere in the ecosystem except for MoveIt and I believe Gazebo. It's easier this way to support many distributions and make sure all changes are reflected in the future. 

Feel free to steal, that was just an afternoon of toying around. If you come up with any more automations, let me know!